db/migrate/20201029160139_add_state_to_external_pull_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToExternalPullRequest < ActiveRecord::Migration[6.0]
+  def change
+    add_column :external_pull_requests, :state, :string
```

Comment:
Instead of it being just a string, we could add a type to the DB and an enum to the code, like we did with `Events::PullRequest` or `Metric`